routes/post.go

```diff
@@ -207,8 +208,8 @@ func (fes *APIServer) _postEntryToResponse(postEntry *lib.PostEntry, addGlobalFe
 		LikeCount:                      postEntry.LikeCount,
 		DiamondCount:                   postEntry.DiamondCount,
 		CommentCount:                   postEntry.CommentCount,
-		RepostCount:                   postEntry.RepostCount,
-		QuoteRepostCount:              postEntry.QuoteRepostCount,
+		RepostCount:                    postEntry.RepostCount,
```

Comment:
We should add back RecloutCount and QuoteRecloutCount to maintain backwards compatibility

Reply:
Ty!